packages/config/src/environment.ts

```diff
@@ -1,8 +1,8 @@
-import resolve from 'resolve';
-import path from 'path';
-import escapeStringRegexp from 'escape-string-regexp';
-import type ts from 'typescript';
-import SilentError from 'silent-error';
+import * as path from 'node:path';
+import resolve = require('resolve');
+import escapeStringRegexp = require('escape-string-regexp');
```

Comment:
…so *not* fancy. 😭 I hate `= require()` so much. It’s definitely the right move here, of course, but 😭 😭 😭